base/shared/src/main/scala/scalaz/data/Liskov.scala

```diff
@@ -0,0 +1,177 @@
+package scalaz
+package data
+
+import Prelude.<~<
+import scalaz.typeclass.{IsContravariant, IsCovariant}
+
+/**
+  * Liskov substitutability: A better `<:<`.
+  *
+  * `Liskov[A, B]` witnesses that `A` can be used in any negative context
+  * that expects a `B`. (e.g. if you could pass an `A` into any function
+  * that expects a `B`.)
+  *
+  * @see [[<~<]] `A <~< B` is a type synonym to `Liskov[A, B]`
+  */
+sealed abstract class Liskov[-L, +H >: L, -A >: L <: H, +B >: L <: H] private[Liskov]() { ab =>
```

Comment:
It's already sealed, so there's no point to a private constructor `sealed abstract class Liskov[-L, +H >: L, -A >: L <: H, +B >: L <: H] private[Liskov]() `

Reply:
Fixed.